src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Can't authenticate the user" + e);
```

Comment:
In the theory we saw, here used the "error level" log. But from my point of view, we should use the "warn level", because that's a common mistake by user and it don't stops or put down a whole logic/server. 
But let's give mentors a decision.

Reply:
Ohh, yes, it's my bad, I didn't spot it/ You are right!